polaris-react/src/components/ActionList/ActionList.scss

```diff
@@ -96,12 +96,9 @@ $indented-item-width: calc(100% - #{$indented-item-margin});
   &.disabled {
     background-image: none;
     color: var(--p-color-text-disabled);
+    cursor: default;
 
-    #{$se23} & {
-      cursor: default;
-    }
-
-    #{$se23} &:hover {
```

Comment:
* [This PR](https://5d559397bae39100201eedc1-tehbqainqm.chromatic.com/?path=/story/all-components-actionlist--with-an-icon-and-a-suffix)
* [Production](https://storybook.polaris.shopify.com/?path=/story/all-components-actionlist--with-an-icon-and-a-suffix&globals=polarisSummerEditions2023:true)

<img width="752" alt="Screenshot 2023-08-25 at 1 45 50 PM" src="https://github.com/Shopify/polaris/assets/20652326/756434a8-8545-4ff2-9c34-2113a3469272">